docs/user-guide/data-structures/data-structures.ipynb

```diff
@@ -327,25 +327,6 @@
     "del da.attrs['aux']"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Distinction between dimension coords and non-dimension coords, and coords and attrs\n",
```

Comment:
Are dimension coords used for anything except plotting at this point?

There should at least be an explanation in plopp about how it determines axis labels.

Reply:
> Are dimension coords used for anything except plotting at this point?

I could not really think of anything, and this is why I simply removed the paragraph.

Reply:
ok